vignettes/git_usage.Rmd

```diff
@@ -100,7 +100,7 @@ Once you have completed the Pull Request you will see all committed changes are
 - The reviewer must ensure that the function follows the programming strategy recommendations. 
 - Any comment/question/discussion must be addressed and documented in GitHub before the Pull Request is merged and the issue closed.
 
-Once the review is completed, and the Pull Request is merged, the branch must be removed: click **Delete branch** in the Pull Request.
+Once the review is completed, and the Pull Request is merged, the branch must be removed: the reviewer will click **Delete branch** in the Pull Request.
```

Comment:
I don't think it's clear who merges the PR. Should be the same person that deletes the branch afterwards.

Reply:
agree - made clearer now that reviewer does both